index.html

```diff
@@ -465,18 +481,18 @@ <h6 class="footer-content">C-BULA</h6>
                 <!-- Links -->
                 <h6 class="footer-shops">Sklepy</h6>
                 <hr class="deep-purple accent-2 mb-4 mt-0 d-inline-block mx-auto" style="width: 60px;">
-                <p>
+                <span>
```

Comment:
tu tez Marta moze miec problemy z CSS jezeli sie zmienia na span. Warto ja zapytac o to.